src/main/resources/db/migration/db.feedback.migration/V202409231227__create_feedback_and_identified_feedback_response.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE feedback (
+    feedback_id BIGSERIAL primary key,
+    company_id BIGINT NOT NULL,
+    department VARCHAR(30),
+    feedback VARCHAR(200) NOT NULL,
```

Comment:
if the table name is feedback, maybe it's better to call this field in another name

Reply:
the table name is "identified_feedback_response"... feels ok?

Reply:
changed to comment